### DISCUSSION TITLE
Sveltekit using @urql/exchange-multipart-fetch

### DISCUSSION THREAD

**Discussion by arperyan** [urql-graphql/urql#1499] [Q&A] [answered]
Hi,

I am uploading images in my sveltekit project and read that I need to install @urql/exchange-multipart-fetch. I am now getting the following error when using the multipartFetchExchange: 

Error when evaluating SSR module /node_modules/@urql/exchange-multipart-fetch/node_modules/extract-files/lib/ReactNativeFile.js:
ReferenceError: module is not defined
    at /node_modules/@urql/exchange-multipart-fetch/node_modules/extract-files/lib/ReactNativeFile.js:3:1
    at instantiateModule (/home/arpeqlik/dev/sveltekit-sickfits/node_modules/vite/dist/node/chunks/dep-efe32886.js:68893:166)

Any help would be greatly appreciated?

Using: 
"@urql/exchange-multipart-fetch": "^0.1.11",
"@urql/svelte": "^1.2.0",
"extract-files": "^9.0.0",        
"urql": "^2.0.1"

**Comment by kitten** [comment, marked as answer]
`extract-files` is currently a mixed module of CommonJS and ESM so it'll need some special help from bunglers to be transpired. I'm not quite sure what Vite provides to support these kinds of packages, but I'd start asking there whether it's fixable
